proto/uber/cadence/admin/v1/service.proto

```diff
@@ -435,3 +447,33 @@ message DynamicConfigFilter {
 	string name = 1;
 	api.v1.DataBlob value = 2;
 }
+
+// Global
+message GetGlobalIsolationGroupsRequest{}
+
+message GetGlobalIsolationGroupsResponse{
+    repeated IsolationGroupConfiguration isolation_groups = 1;
```

Comment:
should `repeated` be removed?

Reply:
yes, thank you, I didn't notice I'd done that twice